elements/howto-tabs/howto-tabs.js

```diff
@@ -284,24 +311,81 @@
       // changes its role to `tab`.
       this.setAttribute('role', 'tab');
```

Comment:
oh right I thought we had stopped doing that after talking with Brian Kardell... Hm, we should chat about this again during the cleanup phase.